### ISSUE TITLE
There are no man pages for doto

### ISSUE THREAD

**Issue by Dbz** [tksfz/doto2#6]
If you just `doto` in the command line, then it shows you the command options, but it would be awesome if there was documentation for `man doto`

**Comment by Dbz** [comment]
This is going to become more and more important over time, so even starting with something simple is good. People will want to know what a task vs. thread vs. event is. As your features get more complex (repo syncing), man pages will be the best place for that documentation. Plus I am willing to help contribute to them once they exist.

**Comment by tksfz** [comment]
I agree with this, but I need to write some documentation first and put it say online somewhere first I think. I was going to write a short tutorial first.

**Comment by Dbz** [comment]
I would start with the man pages first personally. It'll have examples that you can put together into a short tutorial, and you can put that on your github wiki and link to it via the readme. Or if you want to get fancier, get a domain name for it.
